config/environments/test.rb

```diff
@@ -14,7 +14,7 @@
 ENV['BASIC_AUTH_LOGIN_PASSWORD'] = 'test1234'
 ENV['TELEGRAM_BOT_API_KEY'] = 'TOKEN'
 ENV['TELEGRAM_BOT_USERNAME'] = 'TestingBot'
-ENV['HUNDRED_EYES_PROJECT_NAME'] = '50survivors'
+ENV['HUNDRED_EYES_PROJECT_NAME'] = 'TestingProject'
```

Comment:
I think `100eyes` (or even `Unbenanntes Projekt`) would be a better default than `TestingProject`, especially for the future, when real clients will actually use the software.

Reply:
It's `test` environment, clients won't see that.